lib/otel-instrumentation.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const { getNodeAutoInstrumentations } = require('@opentelemetry/auto-instrumentations-node')
+const openTelemetry = require('@opentelemetry/sdk-node')
+
+module.exports = function startTracing({ enableTracing }) {
```

Comment:
I'm not familiar with the otel sdk, but it seems to me that this interface doesn't allow for custom instrumentation to be set, am I right?

How would it be possible for a developer using the lib to implement their own?

Reply:
Not at the moment. Those are some tests, once the instrumentation is successful I think about possible extensibility options.

Reply:
At the moment it is not possible to add new instrumentation to the sdk, but it is always possible to add new span manually (you can see [this example](https://github.com/open-telemetry/opentelemetry-js/blob/main/examples/http/server.js)). If in the future we would add some custom instrumentation, we should think about it. Maybe passing a file/folder with a specific lc39 option